EIPS/eip-2384.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: 2384
+title: Difficulty Bomb Delay
+author: Eric Conner (@econoar)
+type: Standards Track
```

Comment:
Please add a `discussions-to` URL, such as a github issue here or a FEM discussion.